qemu/tests/change_netkvm_params.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from virttest import utils_test
+from virttest import error_context
+from virttest import utils_net
+from virttest.utils_windows import virtio_win
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    The rx/tx offload checksum test for windows
+    1) start vm
+    2) set the netkvm driver parameter value
+    3) get the value and compare to target value
+    4) start ping test to check nic avaliability
+
+    param test: the test object
+    param params: the test params
+    param env: test environment
+    """
+
+    def start_test(param_name, param_value):
```

Comment:
better to rename this function something like `netkvm_change_param_value_test` to reflect its functionality.